src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -26,10 +26,21 @@ private string GetDebuggerDisplay()
         internal const int ListKind = 1;
 
         private readonly ushort _kind;
-        protected NodeFlags flags;
-        private byte _slotCount;
+        private NodeFlagsAndSlotCount _nodeFlagsAndSlotCount;
         private int _fullWidth;
```

Comment:
if we wanted, we could likely grab more bits off of this as well.  For example, we could say the max file size was 256MB, giving us 4 more bits here to store data.